apps/examples/webclient/webclient_main.c

```diff
@@ -253,16 +253,15 @@ int webclient_init_request(void *arg, struct http_client_request_t *request)
 		goto exit;
 	}
 
-	if (!strcmp(argv[1], "GET")) {
+	if (!strncmp(argv[1], "GET", 3)) {
```

Comment:
It seems strncmp("GETS","GET", 3) returns 0.
Is this intended?

Reply:
http(s) has four methods: GET, PUT, POST, DELETE, but GETS.

Reply:
Then, It should be if (!strncmp(argv[1], "GET", 4)), so only "GET" can enter the if statement

Reply:
I modify this as you mentioned.